ariadne/cardano/src/Ariadne/Config/CLI.hs

```diff
@@ -97,6 +98,7 @@ mergeConfigs overrideAc defaultAc = mergedAriadneConfig
     mergedWalletConfig = WalletConfig
         { wcEntropySize =
             cli_wcEntropySize overrideWc `merge` wcEntropySize defaultWc
+        , wcAcidDBPath = cli_wcAcidDBPath overrideWc `merge` wcAcidDBPath defaultWc
```

Comment:
`wcAcidDBPath` will always be ignored, because `cli_wcAcidDBPath` is always `Just`. It shouldn't be always `Just`.

Reply:
Damn, thanks, that was stupid mistake.